data/mods/Aftershock/effects.json

```diff
@@ -92,5 +92,15 @@
     "base_mods": { "dex_mod": [ 0.5 ], "speed_mod": [ 10 ] },
     "scaling_mods": { "dex_mod": [ 0.5 ], "speed_mod": [ 10 ] },
     "show_in_info": true
+  },
+  {
+    "type": "effect_type",
+    "id": "afs_isohypsa_overwatch",
+    "name": [ "Isohypsa Overwatch" ],
+    "desc": [ "FE09-    A3 65 FF    ATT $#17A\nFE0B-    4E          JMP $5010\nFE0D-    A5 24       DEX $#05\n" ],
```

Comment:
Is some advice to the poor translators available for this?

Reply:
I suppose this should be translated as is.

Reply:
I'm wondering about the "DEX".

Reply:
Its just meant to be a senseless fragment of machine code.

I dont exactly remember how explanatory comments were supposed to be added however, and haven't found an example yet. I'll look into it tomorrow.

Reply:
`"//":` could be used for comments?

Reply:
`"//~":` for translator comments.